artemis-distribution/pom.xml

```diff
@@ -287,4 +287,34 @@
       </plugins>
    </build>
 
+   <profiles>
+      <profile>
+         <id>release</id>
```

Comment:
why just on the release?

if I generate a binary, it would be nice to have the HTMLs as well, not just as part of the release.

Reply:
I left in the release profile as it was before, but I haven't any concerns to change it.